packages/auth/tsup.config.ts

```diff
@@ -0,0 +1,27 @@
+import { spawnSync } from "child_process";
+import { defineConfig } from "tsup";
+
+export default defineConfig({
+    entry: ["./src/**/*.ts?(x)", "!./src/types.d.ts", "!./src/__tests__/*"],
+    splitting: true,
+    clean: true,
+    dts: false,
+    format: "esm",
+    tsconfig: "./tsconfig.json",
+    loader: {
```

Comment:
This has the same config as components it seems, making the compiler look for fonts to add to the bundled code. I think you can copy the same tsup config as navigation (?) uses since we already looked at that.

Reply:
Fixed!